ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.6.yaml

```diff
@@ -0,0 +1,102 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+releases:
+  intermediate47:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.7"
+  intermediate48:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.8"
+  intermediate49:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.9"
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+  intermediate411:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.11"
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.6"
+  target:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.12.0-0
+        upper: 4.13.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: azure-ipi-fips-p2-f30
+  cron: 36 18 24 * *
```

Comment:
Perhaps we can initial the date to be `36 18 3 * *` so that the job will be tested and we'll have the results in RP in a  short time, and then use a follow up PR to batch update them back.

Reply:
Updated. thanks for the suggestion!